designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRoot.kt

```diff
@@ -371,6 +372,7 @@ fun SquooshRoot(
             isRoot,
             VariableState.create(),
             overlays,
+            LocalContext.current,
```

Comment:
I was planning to use LocalContext.current directly in the Utils  instead of passing it down but it requires the method to be a @Composable. The only reason I didn't do that is, I am not sure if I should change the Squoosh methods like `squooshComputeTextInfo` and `resolveVariantsRecursively` to @Composable.

Reply:
I can not think of a better way to get the app context in a non composable function. I will leave it passed as parameters at this point.